.test-infra/jenkins/job_LoadTests_GBK_Java.groovy

```diff
@@ -0,0 +1,262 @@
+                                              "hotKeyFraction": 1
+                                            }
+                                       """.trim().replaceAll("\\s", ""),
+                            fanout                : 1,
+                            iterations            : 4,
+                            maxNumWorkers         : 5,
+                            numWorkers            : 5,
+                            autoscalingAlgorithm  : "NONE",
+                            streaming             : isStreaming
+                    ]
+            ],
+            [
+                    title        : 'Load test: reiterate 4 times 2MB values',
+                    itClass      : 'org.apache.beam.sdk.loadtests.GroupByKeyLoadTest',
+                    runner       : CommonTestProperties.Runner.DATAFLOW,
+                    jobProperties: [
+                            project               : 'apache-beam-testing',
+                            appName               : "load_tests_Java_Dataflow_${mode}_GBK_7",
+                            tempLocation          : 'gs://temp-storage-for-perf-tests/loadtests',
+                            publishToBigQuery     : true,
+                            bigQueryDataset       : 'load_test',
+                            bigQueryTable         : "java_dataflow_${mode}_GBK_7",
+                            sourceOptions         : """
+                                            {
+                                              "numRecords": 20000000,
+                                              "keySizeBytes": 10,
+                                              "valueSizeBytes": 90,
+                                              "numHotKeys": 10,
+                                              "hotKeyFraction": 1
+                                            }
+                                       """.trim().replaceAll("\\s", ""),
+                            fanout                : 1,
+                            iterations            : 4,
+                            maxNumWorkers         : 5,
+                            numWorkers            : 5,
+                            autoscalingAlgorithm  : "NONE",
+                            streaming             : isStreaming
+                    ]
+            ]
+    ]
+}
+
+def streamingLoadTestJob = { scope, triggeringContext ->
+  scope.description('Runs Java GBK load tests on Dataflow runner in streaming mode')
+  commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 240)
+
+  for (testConfiguration in loadTestConfigurations('streaming', true)) {
+      testConfiguration.jobProperties << [inputWindowDurationSec: 1200]
+    loadTestsBuilder.loadTest(scope, testConfiguration.title, testConfiguration.runner, CommonTestProperties.SDK.JAVA, testConfiguration.jobProperties, testConfiguration.itClass, triggeringContext)
+  }
+}
+
+CronJobBuilder.cronJob('beam_LoadTests_Java_GBK_Dataflow_Streaming', 'H 12 * * *', this) {
```

Comment:
Please change them to be run on a different hour (eg. 2 hours later) to avoid extensive usage and reduce "noisy neighbors" problem.

Reply:
ok!